content/node-runners/payout-guide.md

```diff
@@ -75,3 +75,56 @@ On mobile, the process is exactly the same. However, you might find it a bit har
 
 And that’s it. You’ve learned how to add your tokens and are a step closer to plunging into the decentralized ecosystem!
 
+
+## Using MetaMask with Polygon MYST
+
+Polygon MYST can be added to your MetaMask with a few simple clicks.
+
+### Link MetaMask to the Polygon Network
+
+1. Open MetaMask console and switch the connected blockchain (network) by clicking on the tab saying Ethereum Mainnet, we need to add the Matic network.
+2. Scroll down until you find Custom RPC.
+3. Enter in the Matic Mainnet settings as follows:
+
+> Network Name: Matic Mainnet
+
+> New RPC URL: https://rpc-mainnet.maticvigil.com/
+
+> Chain ID: 137
+
+> Currency Symbol (optional): MATIC
+
+> Block Explorer URL (optional): https://explorer.matic.network/
+
+<div style="text-align:center">
+  <img src="../images/platforms/mp1.png" alt="Polygon" class="screenshot"/>
+</div>
+
+
+You have now connected to the Matic mainnet with your Metamask Wallet! You can now interact with Polygon tokens like $MYST while connected to Polygon. Note that transactions on the Polygon network requires its native token MATIC which will be used as gas fee. You can get Matic tokens from most exchanges.
+
+### Adding MYST token to MetaMask on the Polygon Mainnet
+
+Follow our previous guide - > Adding MYST token to MetaMask on the Ethereum Mainnet. Please note that recently created Polygon (Matic) Mainnet blockchain should be used instead of the Ethereum Mainnet!
```

Comment:
I think that might not work as on MATIC we got a different address for the token: https://polygonscan.com/token/0x1379e8886a944d2d9d440b3d88df536aea08d9f3